aws/resource_aws_sagemaker_notebook_instance_test.go

```diff
@@ -0,0 +1,320 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+const sagemakerTestAccSagemakerNotebookInstanceResourceNamePrefix = "terraform-testacc-"
+
+func init() {
```

Comment:
It looks like you started a sweeper but didn't finish it. We can just remove this unless you wanted to finish it